index.js

```diff
@@ -17,8 +17,24 @@ const EMAIL_PERMISSION = "alexa::profile:email:read";
 const EMPTY_STATE = 'empty';
 // used to move on to next step
 const TERMINATION_WORD = 'next';
+const helpWords = ['help'];
+const summaryWords = ['summary'];
```

Comment:
Add more words like 'summarize'. Make sure the function i wrote below works to catch all the words.